ngraph/core/include/openvino/core/check.hpp

```diff
@@ -91,7 +91,7 @@ class OPENVINO_API CheckFailure : public Exception {
 // Then, we define the macro NODE_VALIDATION_CHECK as follows:
 //
 // #define NODE_VALIDATION_CHECK(node, cond, ...) <backslash>
-//     OV_CHECK_HELPER(::ov::NodeValidationFailure, (node), (cond), ##__VA_ARGS__)
+//     OPENVINO_CHECK_HELPER(::ov::NodeValidationFailure, (node), (cond), ##__VA_ARGS__)
```

Comment:
can we have better name like `OPENVINO_ASSERT`?

Reply:
Fixed